.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "pwndbg",
+	"build": {
+		"context": "..",
+		"dockerfile": "../Dockerfile"
+	},
+	"runArgs": [
+		"--cap-add=SYS_PTRACE",
+		"--security-opt",
+		"seccomp=unconfined"
```

Comment:
Why do we need to disable seccomp? Also, if we are here can we add `--security-opt no-new-privs:true` (or sth like that) and `--cap-drop=ALL` before adding PTRACE?

Reply:
@disconnect3d Fixed, thanks.